cpp/src/randomforest/randomforest.cu

```diff
@@ -341,6 +347,102 @@ std::vector<unsigned char> save_model(ModelHandle model) {
   return bytes_info;
 }
 
+void check_concat_tl_mod(ModelHandle concat_tree_handle,
+                         std::vector<ModelHandle*> treelite_handles) {
+  size_t concat_forest;
+  size_t single_handle;
```

Comment:
It'll be clearer if this is a local variable in the loop blow and has a different name to convey that it represents number of trees in the current forest.